app/config/settings.py

```diff
@@ -128,8 +128,13 @@
 FLATPAGE_ABOUT_URL = os.environ.get("FLATPAGE_ABOUT_URL", "/about/")
 
 # Costs (in US dollar cents)
-CHALLENGES_STORAGE_COST_CENTS_PER_TB_PER_YEAR = os.environ.get(
-    "CHALLENGES_STORAGE_COST_CENTS_PER_TB_PER_YEAR", 4000
+# based on 0.023 / GB / month S3 standard pricing
+CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR = os.environ.get(
+    "CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR", 27600
+)
```

Comment:
You will need to cast the environment variables to an `int` as they always return a string, check the usage elsewhere (CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR, CHALLENGES_ECR_STORAGE_COST_CENTS_PER_TB_PER_YEAR, CHALLENGES_COMPUTE_COST_CENTS_PER_HOUR)

```suggestion
CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR = int(os.environ.get(
    "CHALLENGES_S3_STORAGE_COST_CENTS_PER_TB_PER_YEAR", 27600
))
```